src/scala/main/ly/stealth/mesos/kafka/scheduler/http/api/TopicApi.scala

```diff
@@ -59,6 +59,27 @@ trait TopicApiComponentImpl extends TopicApiComponent {
       @DefaultValue("*") @BothParam("topic") expr: String
     ) = list(expr)
 
+    @Path("delete")
+    @POST
+    @Produces(Array(MediaType.APPLICATION_JSON))
+    def deletePost(
+      @BothParam("topic") topicExpr: String
```

Comment:
For new APIs we can/should use `@FormParam` instead of `BothParam`.  `BothParam` is only a hack for backwards compatibility, where the old client would send everything in the query string.